fluentd/lib/fluent-plugin-collected/lib/fluent/plugin/in_collected_tail_monitor.rb

```diff
@@ -94,11 +94,11 @@ def update_monitor_info
           if pe.read_inode == old_inode && pe.read_pos >= old_pos
             # Same file, has not been truncated since last we looked. Add the delta.
             @log.trace "delta bytes #{pe.read_pos}  #{old_pos}"
-            @metrics[:total_bytes_collected].increment(label, pe.read_pos)
+            @metrics[:total_bytes_collected].increment(labels: label, by: pe.read_pos)
```

Comment:
this needs to be `by: pe.read_pos - old_pos`, see this code pre https://github.com/ViaQ/logging-fluentd/pull/21